data/formative-questionnaires/personal-social-and-emotional-development.yml

```diff
@@ -220,8 +219,10 @@ personal-social-and-emotional-development:
 
```

Comment:
the spacing on the assessment summary hasn't rendered properly

Reply:
I think I've fixed this :)